src/lib/templateResource.ts

```diff
@@ -0,0 +1,40 @@
+type ResourceReducer =
+  | ((resource: string, config: Config) => string)
+  | ((resource: string) => string);
+
+// Removes all spaces from string, this allows the other reducers to have more
+// reliable match patterns as they don't need to be concerned about whitespace.
+function removeWhitespace(resource: string): string {
+  return resource.replace(/\s/g, "");
```

Comment:
Would url/resources come in with `\s` character or do they have the potential to be encoded as `%20` ?

Reply:
The API does not URL encode input.